contentcuration/contentcuration/settings.py

```diff
@@ -242,11 +242,11 @@
 ugettext = lambda s: s
 LANGUAGES = (
     ('en', ugettext('English')),
-    ('es', ugettext('Spanish')),
+    # ('es', ugettext('Spanish')),
     # ('ar', ugettext('Arabic')), # Uncomment when we have translations
-    ('es-es', ugettext('Spanish - Spain')),
+    # ('es-es', ugettext('Spanish - Spain')),
     ('es-mx', ugettext('Spanish - Mexico')),
```

Comment:
What happens if someone requests with the `es` language code?

Reply:
Django falls back to `es-mx`, as it knows that's the closest thing we have.

Otherwise, you get the behavior in #405